WcaOnRails/app/webpacker/components/Translators/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useMemo } from 'react';
+import { Container, Grid, Header } from 'semantic-ui-react';
+import useLoadedData from '../../lib/hooks/useLoadedData';
+import { apiV0Urls } from '../../lib/requests/routes.js.erb';
+import { groupTypes } from '../../lib/wca-data.js.erb';
+import Loading from '../Requests/Loading';
+import Errored from '../Requests/Errored';
+import I18n from '../../lib/i18n';
+import UserBadge from '../UserBadge';
+
+export default function Translators() {
+  const {
+    data: translators, loading, error,
+  } = useLoadedData(apiV0Urls.userRoles.listOfGroupType(groupTypes.translators));
+  const groupedTranslators = useMemo(
+    () => Object.groupBy(translators || [], (role) => role.group.id),
+    [translators],
+  );
+  const translatorGroups = useMemo(
+    () => Object.keys(groupedTranslators).map((key) => groupedTranslators[key][0].group),
+    [groupedTranslators],
+  );
+
+  if (loading) return <Loading />;
+  if (error) return <Errored />;
+
+  return (
+    <Container fluid>
+      <Header as="h2">{I18n.t('page.translators.title')}</Header>
+      {translatorGroups.map((group) => (
```

Comment:
I think I can do something here, but I'll attend this once the other comment is resolved (if it is still relevant then) because any action to that might make this deprecated.

Reply:
Changed.

Reply:
No need to change anything, but you could have used `Object.entries` to get both the key (group id) and the value (group). Instead of using `groupedTranslators[groupId]` inside the map to get the group.

Reply:
Object.entries is better, used it now. Thank you. :-)